lighthouse-core/lib/rect-helpers.js

```diff
@@ -208,13 +204,15 @@ function getRectAtCenter(rect, centerRectSize) {
 /**
  * @param {LH.Artifacts.Rect} rect
  */
+/* istanbul ignore next */
```

Comment:
same with this and `getLargestRect`, which are called from the `tap-targets.js` audit?